src/music/music.module.ts

```diff
@@ -0,0 +1,13 @@
+import { Module } from '@nestjs/common';
```

Comment:
Module Name Should Be In Plural Also Controller Service And Repository Should Be Plural

Reply:
> Module Name Should Be In Plural Also Controller Service And Repository Should Be Plural

mravlobitshi anu ase : MusicControllers?